Storage/SQL/SQLiteRemoteClientsAndTabs.swift

```diff
@@ -381,88 +311,69 @@ open class SQLiteRemoteClientsAndTabs: RemoteClientsAndTabs {
     }
 
     open func deleteCommands() -> Success {
-        var err: NSError?
-        _ = db.transaction(&err) { connection, _ in
-            if let error = connection.executeChange("DELETE FROM \(TableSyncCommands)", withArgs: [] as Args) {
-                print(error.description)
-                err = error
-                return false
-            }
-            
-            return true
+        return db.transaction { connection -> Void in
+            try connection.executeChange("DELETE FROM \(TableSyncCommands)", withArgs: [] as Args)
```

Comment:
We should probably revisit single-statement transactions. They don't need to be wrapped in `BEGIN TRANSACTION`, and adding a `db.transact(sql: String, withArgs: Args)` variant would make a bunch of these functions into one-liners.